scripts/drush8.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This is cool, thanks.  FYI, I'm starting to look at packaging Drush 7 & 8 as RPMs, so hopefully this becomes easier.